docs/developer/.scripts/33_render_status.py

```diff
@@ -371,7 +371,7 @@ def render_config_validation_progress():
 
 
 def render_hatch_migration_progress():
-    valid_checks = sorted(get_valid_integrations())
+    valid_checks = sorted(get_valid_checks())
```

Comment:
Hmm, so the difference between an "integration" and a "check" is whether a manifest or a version file is present?

And we don't want to migrate "integrations" to hatch?

Reply:
All integrations have a manifest file. Integration with no tox file don't need to be migrated to hatch

Reply:
Aha, and we're using the term "check" to refer to integrations that have tox/hatch environments?

Reply:
That are python integration, so they have a tox/hatch envs yes

Reply:
https://datadoghq.dev/integrations-core/faq/faq/#integration-vs-check